sass/utils/_variables.scss

```diff
@@ -31,7 +31,7 @@ $transition-quick: 0.12s;
 
 // Elevations
 $elevation-1: 0 2px 3px 0 alpha-color($black, .08);
-$elevation-2: 0 4px 6px 0 alpha-color($black, .012);
+$elevation-2: 0 4px 6px 0 alpha-color($black, .12);
```

Comment:
I followed the discussion above on this, but I'm worried about seeing this change in isolation. @matthewbirtch, can you help provide guidance here?

Reply:
@lieut-data `.12` was incorrectly coded as `.012`  value previously and had just never been caught.

Reply:
@lieut-data I agree with @matthewbirtch, it was incorrectly coded, and is not used anywhere.

Reply:
Thanks, @matthewbirtch & @asaadmahmood -- should we go ahead and ask @TheDarkestDay to change all of these `elevation-*` values to be correct? Otherwise I fear regressing here by some future "fix" to what /now/ looks like a bug.

Reply:
I don't mind changing them in another PR, or actually having it done in this one. I'm fine either way. @lieut-data

Reply:
Thanks @asaadmahmood -- to confirm, should `$elevation-1` have an alpha component of `.08` and all of `$elevation-2 .. $elevation-6` have an alpha component of `.12`?

Reply:
@asaadmahmood (or @matthewbirtch), can you confirm my interpretation of this thread and the final `$elevation-*` values above?